src/odemis/util/linalg.py

```diff
@@ -157,3 +157,53 @@ def qlp(a, mode='reduced'):
     """
     q, r = qrp(numpy.flip(a), mode)
     return numpy.flip(q), numpy.flip(r)
+
+
+def fit_plane_lstsq(coordinates):
```

Comment:
If we keep these functions, they will need some unit tests.

Reply:
yes, Karishma is working on that